doc/whats_new.rst

```diff
@@ -93,6 +93,8 @@ Bug
 API
 ~~~
 
+- Channels with unknown locations are now assigned position ``[np.nan, np.nan, np.nan]`` instead of ``[0., 0., 0.]``, by `Eric Larson`_
```

Comment:
isn't this the convention we follow everywhere to use [0, 0, 0]? What is the default in `create_info`?

Reply:
oh never mind, I thought this was a fix to EEGLAB but I see you made the change everywhere